bin/dsn.cmake

```diff
@@ -296,7 +296,11 @@ function(dsn_setup_thirdparty_libs)
     set(DEFAULT_THIRDPARTY_LIBS ${THRIFT_LIB} fmt::fmt CACHE STRING "default thirdparty libs" FORCE)
 
     # rocksdb
-    list(APPEND CMAKE_MODULE_PATH "${DSN_PROJECT_DIR}/thirdparty/src/pegasus-rocksdb-6.6.4-compatible/cmake/modules")
+    file(GLOB ROCKSDB_DEPENDS_MODULE_PATH ${DSN_PROJECT_DIR}/thirdparty/src/*/cmake/modules)
```

Comment:
```suggestion
    file(GLOB ROCKSDB_DEPENDS_MODULE_PATH ${DSN_PROJECT_DIR}/thirdparty/src/pegasus-rocksdb-6.6.4-compatible/cmake/modules)
```

Reply:
use regex to find modules, so we don't need to modify this when we change the rocksdb src name(e.g. dump version or rename)